rotkehlchen/chain/ethereum/modules/yearn/vaults.py

```diff
@@ -556,18 +568,24 @@ def _get_vault_withdraw_events(
             )
             tx_hash = deserialize_evm_tx_hash(withdraw_event['transactionHash'])
             tx_receipt = self.ethereum.get_transaction_receipt(tx_hash)
+            if tx_receipt is None:
```

Comment:
In `_query` you added a check so that for receipts and transactions pruned nodes are not queried. So is there a need to handle an empty receipt here?

Reply:
An empty receipt is possible since a `tx_hash` might be valid but does not exist on-chain.

Reply:
Yeah, I mean that this tx hash is retrieved from chain, so it is not something user input, and thus I would say we can rely on it's existence.

But anyway, I think having an extra check won't hurt.